src/rcc.rs

```diff
@@ -25,8 +25,11 @@ impl RccExt for RCC {
             apb2: APB2 { _0: () },
             bdcr: BDCR { _0: () },
             csr: CSR { _0: () },
+            crrcr: CRRCR { _0: () },
             cfgr: CFGR {
                 hclk: None,
+                hsi48: false,
+                lsi: true,
```

Comment:
Could this be default false please, the LSI is only required for the for the RTC (none critical)

Reply:
Sure, it's just that the existing implementation turns it on, so off is a breaking change and technically you will have to increment the minor version :)

Reply:
That's okay, it seems before a rust crate hits 1.0, the trend is to treat  0.a.b versions like a.b.0 ones for the sake of compatibility with changing versions. So I will just bump the patch version number.